main.js

```diff
@@ -0,0 +1,24 @@
+function daysInYear(year){
+  try {
+
+    if(!Number.isInteger(year)) {
+    throw new Error (`exception`);
+  } 
+    let start = new Date(year,0, 0);
+    let end = new Date(year,11,31);
+    let diff = end-start;
+    let days = diff/(1000*60*60*24);
+    return days;
+
+  } catch(e) {
+    return e.message;
```

Comment:
Based on the requirements this method should `it should throw an exception`. Currently it doesn't throw an exception.